src/jshint.js

```diff
@@ -3014,17 +3009,17 @@ var JSHINT = (function() {
 
   /**
    * @param {Object} [options]
-   * @param {token} [options.name] The identifier belonging to the function (if
+   * @param {string} [options.name] The identifier belonging to the function (if
    *                               any)
-   * @param {boolean} [options.statement] The statement that triggered creation
+   * @param {token} [options.statement] The statement that triggered creation
    *                                      of the current function.
    * @param {string} [options.type] If specified, either "generator" or "arrow"
    * @param {token} [options.loneArg] The argument to the function in cases
    *                                  where it was defined using the
    *                                  single-argument shorthand
    * @param {bool} [options.parsedOpening] Whether the opening parenthesis has
    *                                       already been parsed
-   * @param {token} [options.classExprBinding] Define a function with this
+   * @param {string} [options.classExprBinding] Define a function with this
```

Comment:
The changes to this comment are good, but they subvert the formatting convention. Could you also update the indentation of the text in the subsequent lines?